middleware/proxy/README.md

```diff
@@ -15,6 +15,7 @@ Proxy middleware for [Fiber](https://github.com/gofiber/fiber) that allows you t
 func Balancer(config Config) fiber.Handler
 func Forward(addr string, clients ...*fasthttp.Client) fiber.Handler
 func Do(c *fiber.Ctx, addr string, clients ...*fasthttp.Client) error
+func DomainForward(hostname string, addr string, clients ...*fasthttp.Client) fiber.Handler 
```

Comment:
can you pls add description type blocks above the methods of the readme, so that everyone can see what these methods can be used for

Reply:
I have added a short description to the signature in the readme